package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
-    "gulp": "^4.0.0",
+    "gulp": "^4.0.2",
```

Comment:
Please don't update dependencies. So please revert `package.json` and `package-lock.json`.

Reply:
Fixed. This was an accident. I didn't realize that `npm install` would update the packages.